rules/config

```diff
@@ -300,3 +300,6 @@ GZ_COMPRESS_PROGRAM ?= gzip
 
 # SONIC_OS_VERSION - sonic os version
 SONIC_OS_VERSION ?= 11
+
+# ENABLE_RFS_SPLIT - enable 2-stage installer build
+ENABLE_RFS_SPLIT_BUILD ?= n
```

Comment:
>ENABLE_RFS_SPLIT_BUILD [](http://example.com/codeflow?start=0&length=22)

Since it is an optimization, no need to enable it by config, just enable by default.

Reply:
I'd suggest to consider keeping the flag and having it on by default.
In this case, if there are any issues, we can just disable it and have the build working as it is now.

@qiluo-msft @xumia @Kalimuthu-Velappan what do you think?

Reply:
If there are any issues, we can just revert this PR since the PR is small.

Reply:
Ok, done